### ISSUE TITLE
Migrate assertion verdict values in database to pass/fail values

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#735]
This issue is one element of the plan documented in w3c/aria-at#961 to migrate from 3 assertion verdicts (good/no output/incorrect output) to two verdicts (pass/fail).

This change should be made after or at the same time as #733. It is a prerequisite for w3c/aria-at#969.

In the database migrate all assertion verdicts as follows:

* good output = pass
* no output = fail
* incorrect output = fail

**Comment by ccanash** [comment]
Hi @mcking65 , this has been completed
